docs/static/_redirects

```diff
@@ -1 +1,3 @@
 /tutorial /docs/ios/tutorial/tutorial-introduction
+/tutorial/tutorial-create-project /docs/ios/tutorial/tutorial-add-sdk
+/docs/ios/tutorial/tutorial-create-project /docs/ios/tutorial/tutorial-add-sdk
```

Comment:
@trevorblades - do I need both of these redirects or should I only need one or the other to account for a file being renamed?